.github/workflows/ci.yml

```
@@ -15,8 +17,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        persist-credentials: false
     - name: Build Docker image
-      uses: docker/build-push-action@v1
+      uses: docker/build-push-action@3e7a4f6646880c6f63758d73ac32392d323eaf8f # v1
```

Fairly outdated, but should set to the full version:
```suggestion
      uses: docker/build-push-action@3e7a4f6646880c6f63758d73ac32392d323eaf8f # v1.1.2
```